mapie/control_risk/p_values.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from numpy.typing import NDArray
+from typing import Iterable, Union
+from scipy.stats import binom
+
+
+def hoefdding_bentkus_p_value(
```

Comment:
UPD: rename this function `compute_hoefdding_bentkus_p_value`.

Reply:
Done